panel/pane/vtk/__init__.py

```diff
@@ -0,0 +1,9 @@
+
+from .vtk import VTK
+
+try:
+    import vtk
```

Comment:
Large libraries like vtk should not be imported until the VTK pane is actually used.